.github/workflows/nestbuildmatrix.yml

```
@@ -666,6 +659,7 @@ jobs:
               -Dwith-warning=${{ contains(matrix.use, 'warning') && 'ON' || 'OFF' }} \
               -Dwith-boost=${{ contains(matrix.use, 'boost') && 'ON' || 'OFF' }} \
               -Dwith-openmp=${{ contains(matrix.use, 'openmp') && 'ON' || 'OFF' }} \
+              ${{ contains(matrix.use, 'openmp') && '-DOpenMP_ROOT=`brew --prefix libomp`' }} \
```

The use of back ticks is discouraged due to unclear nesting in many situations.
```suggestion
              ${{ contains(matrix.use, 'openmp') && '-DOpenMP_ROOT=$(brew --prefix libomp)' }} \
```